### PR TITLE
WebView send measurements without metric

### PR DESCRIPTION
**Pull request by vito1188**
Change the order of calling end: metric method. (The metric is ended and set to nil before the measurement is ended, so the measurement has no metric)
Set the min time of measurement to 1ms.
Start the URL measurement in loadRequest: method.

Tested on the device(iOS 11)

### CODE REVIEW

File: RPerformanceTracking/Private/_RPTClassManipulator+UIWebView.m [hunk 1]
```diff
@@ -149,7 +151,9 @@ + (void)_swizzleUIWebViewDelegate:(id<UIWebViewDelegate>)delegate
             ((void(*)(id, SEL, id))originalImp)(selfRef, selector, webView);
         }
 
-        endTrackingWithUIWebView(webView);
+        dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(0.1 * NSEC_PER_SEC)), dispatch_get_main_queue(), ^{
+            endTrackingWithUIWebView(webView);
+        });
     };
```
Comment (donnie-jp) [inline RPerformanceTracking/Private/_RPTClassManipulator+UIWebView.m]: Please remove the dispatch_after. It shouldn't be required.

File: RPerformanceTracking/Private/_RPTClassManipulator+UIWebView.m [hunk 2]
```diff
@@ -36,23 +36,25 @@ static void endTrackingWithUIWebView(UIWebView *webView)
...
 
+    uint_fast64_t trackingIdentifier = [objc_getAssociatedObject(webView, _RPT_UIWEBVIEW_TRACKINGIDENTIFIER) unsignedLongLongValue];
     if (trackingIdentifier)
     {
         // Update status code
-        NSURLRequest* request = webView.request;
-        NSCachedURLResponse *urlResponse = [[NSURLCache sharedURLCache] cachedResponseForRequest:request];
-        NSHTTPURLResponse *httpResponse = (NSHTTPURLResponse*)urlResponse.response;
-        if (httpResponse.URL)
+        if (url)
```
Comment (donnie-jp) [inline RPerformanceTracking/Private/_RPTClassManipulator+UIWebView.m]: I don't think we need to check url here. We can just update the status code regardless.

**Review by donnie-jp — CHANGES_REQUESTED** [review]
inline comments

File: Tests/UIWebViewHookTests.m [hunk 3]
```diff
@@ -148,17 +140,6 @@ - (void)testEndMetricCalledOnWebViewDidFailLoadWithError
...
-{
-    id mockTracker = OCMPartialMock(self.trackingManager.tracker);
-    
-    WebViewControllerChild *child = [self childWebViewController];
-    [child.webViewInChild.delegate webViewDidStartLoad:child.webViewInChild];
-    
-    OCMVerify([mockTracker prolongMetric]);
-    [mockTracker stopMocking];
-}
-
```
Comment (donnie-jp) [inline Tests/UIWebViewHookTests.m]: Instead of dropping this test can we replace it with a webViewDidFinishLoad check?

Comment (vito1188) [reply Tests/UIWebViewHookTests.m]: The unit test with a webViewDidFinishLoad has already existed.

**Review by donnie-jp — APPROVED** [review]

**Pull request merged**